evap/staff/importers.py

```diff
@@ -42,13 +50,20 @@ def store_in_database(self):
             user.refresh_login_key()
         return user, created
 
-    def validate(self):
+    def user_already_exists(self):
+        return UserProfile.objects.filter(username=self.username).exists()
+
+    def create_user_profile(self):
```

Comment:
`UserProfile.objects.create(...)` is the method to create objects in the database, so using `create` here is problematic i think. maybe construct, init or instantiate